.github/workflows/chromatic-push.yml

```
@@ -19,7 +19,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v1
         with:
-          node-version: ">=14.16.0 14"
+          node-version: ">=16.16.0 16"
```

Why not target all versions of v16 node?
```suggestion
          node-version: ">=16"
```

---

I decided target the LTS of v16 as wasn't sure there was much point using a version below it. I also locked to v16 as it was previously for v14 so that it doesn't use a v17/v18 node version for now

---

Im fine for us to use `">=16.16.0 16"`, the current active lts.